op-node/service.go

```diff
@@ -206,6 +206,10 @@ Conflicting configuration is deprecated, and will stop the op-node from starting
 			canyon := ctx.Uint64(flags.CanyonOverrideFlag.Name)
 			config.CanyonTime = &canyon
 		}
+		if ctx.IsSet(flags.DeltaOverrideFlag.Name) {
```

Comment:
I see that we have to apply the overrides in two places. Can we extract these `if` statements into a method and call the method from both places? Concerned we'll miss an override in the future.

Reply:
Implemented it as a helper function like Matt suggested, the code-path dedup diff is easier to follow, and I like smaller functions.